localai/cve-2024-2029/README.md

```diff
@@ -0,0 +1,20 @@
+# requirements
+1. install docker
+2. install wget
+
+# secure instance setup
+```bash
+docker run -ti -p 8080:8080 --rm -d --name localaiSecure localai/localai:v2.10.0-ffmpeg-core
+ls whisper || wget wget "https://huggingface.co/ggerganov/whisper.cpp/resolve/main/ggml-tiny.bin?download=true" -O whisper
+docker cp whisper localaiSecure:/build/models/whisper
+docker cp whisper.yaml localaiSecure:/build/models/whisper.yaml
```

Comment:
Please avoid using `docker cp` and mount a volume to make the required file available inside the container. If you don't want non-required files to be accessible from the container create a separate folder (eg. `models/`) and mount that.

```suggestion
whisper || wget wget "https://huggingface.co/ggerganov/whisper.cpp/resolve/main/ggml-tiny.bin?download=true" -O whisper
docker run -ti -p 8080:8080 --rm -d --name localaiSecure -v $(pwd):/build/models localai/localai:v2.10.0-ffmpeg-core
```